awx/ui_next/src/screens/Inventory/InventoryGroupHosts/InventoryGroupHostList.jsx

```diff
@@ -179,25 +202,26 @@ function InventoryGroupHostList({ i18n }) {
         }
       />
       {isModalOpen && (
-        <AlertModal
-          isOpen={isModalOpen}
-          variant="info"
-          title={i18n._(t`Select Hosts`)}
+        <AssociateModal
+          header={i18n._(t`Hosts`)}
+          fetchRequest={fetchHostsToAssociate}
+          isModalOpen={isModalOpen}
+          onAssociate={handleAssociate}
           onClose={() => setIsModalOpen(false)}
-        >
-          {/* ADD/ASSOCIATE HOST MODAL PLACEHOLDER */}
-          {i18n._(t`Host Select Modal`)}
-        </AlertModal>
+          title={i18n._(t`Select Hosts`)}
+        />
       )}
-      {disassociateError && (
+      {(associateError || disassociateError) && (
```

Comment:
you should be able to test just for `error` here (which will equal `associateError || disassociateError` since that's what you gave to the useDismissableError hook)